microsoftgraph/microsoft-graph-explorer
# updates version to v1.7.12

## Overview

Creates build files for version 1.7.12
Fixes [#285](https://github.com/microsoftgraph/microsoft-graph-explorer/issues/285)
Fixes transient authentication issues

## Testing Instructions

* Load app, check to see words on the login button are now 'Sign In'
* AppVersion shows 1.7.12
